### PR TITLE
Prefill docs

### PR DESCRIPTION
**Pull request by tpill90**
- Added a section for slow Battle.net downloads to **common issues** section
- Adding links to prefill apps
- Fixing weird "permalink" text being appended to table of contents, when using Firefox with adblock 
image

### CODE REVIEW

File: _docs/faq.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Other options also involve transparently intercepting HTTP traffic at a network
 
 ## Can I load some content into the cache ahead of my event?
 
-You could look at [zeropingheroes/lancache-autofill](https://github.com/zeropingheroes/lancache-autofill) as an aide to pre-loading popular games into the cache ahead of time to help kick start the cache
+You could look at [steam-lancache-prefill](https://github.com/tpill90/steam-lancache-prefill) and  [battlenet-lancache-prefill](https://github.com/tpill90/battlenet-lancache-prefill) as  aides to pre-load popular games into the cache ahead of time to help kick start the cache.
```
Comment (astrolox) [inline _docs/faq.md:84]: Is there something wrong with the existing link that we'd want to change it?

File: pages/resources.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ Organisers of LAN events and gaming cafe's have been working on the problem of c
 - [Open Source Lan](http://opensourcelan.com/)
 
 ## Cache Priming
-- [ZeroPingHeroes](https://github.com/zeropingheroes/lancache-autofill)
-- [Battlenet-lancache-prefill](https://github.com/tpill90/Battlenet-lancache-prefill)
+- [steam-lancache-prefill](https://github.com/tpill90/steam-lancache-prefill)
```
Comment (astrolox) [inline pages/resources.md:15]: Same as above.

**Review by astrolox — CHANGES_REQUESTED** [review]
Please restore the ZeroPingHeroes links, or provide an explanation why we should switch.

File: _docs/faq.md (hunk 1 above)
Comment (mintopia) [reply _docs/faq.md:84]: As discussed on Discord,. steam-prefill is a much better solution!

**Comment by mintopia** [comment]
All good - will merge.

**Pull request merged**